### PR TITLE
Log activity

### PR DESCRIPTION
**Pull request by dianaboiangiu**

### CODE REVIEW

**Comment by dianaboiangiu** [comment]
Clone option - should we log it as cloning or just as adding?
Data provider detail should distinguish between network and non-network? Right now it logs as " user visited the detail page of data provider x" for both.
Should we log Import, Export?
Should we log Product?

File: insitu/views/action_logging/base_logging.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+class ListLoggingView(GetMethodLoggingView):
+    get_action = 'visited list page of'
+
+
+class DetailLoggingView(GetMethodLoggingView):
+    get_action = 'visited detail page of'
+
+    def get_object_id(self):
+        return self.get_object().id
```
Comment (andreeadima) [inline insitu/views/action_logging/base_logging.py]: New line 😄

File: insitu/views/action_logging/base_logging.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+
+class PostMethodLoggingView(BaseLoggingView):
+    post_action = None
+
+    def post(self, request, *args, **kwargs):
+        response = super().post(request, *args, **kwargs)
+        self.extra = json.dumps(self.get_form().errors)
+        if self.extra:
+            self.post_action = 'tried to create'
```
Comment (andreeadima) [inline insitu/views/action_logging/base_logging.py]: I would have another post_action_failed attribute and overwrite it in the views.

**Comment by andreeadima** [comment]
- would be nice to log as clone
- no diference in detail
- no logging for import/export
- no logging for product

**Comment by dianaboiangiu** [comment]
Updated pull request

**Pull request merged**